cluster/src/test/java/org/apache/iotdb/cluster/log/applier/DataLogApplierTest.java

```diff
@@ -374,4 +376,20 @@ public void testApplyCreateMultiTimeseiresWithPulling() throws MetadataException
     assertTrue(IoTDB.metaManager.getAllStorageGroupPaths().contains(new PartialPath("root.sg2")));
     assertNull(log.getException());
   }
+
```

Comment:
Good Job.

Generally, the testcontainer needs to be added to implement the functions of the cluster edition.

Reply:
thanks, i will add testcontainer